Sources/MapboxNavigation/NavigationViewController.swift

```diff
@@ -233,6 +233,7 @@ open class NavigationViewController: UIViewController, NavigationStatusPresenter
                               routeProgress: navigationService.routeProgress)
         }
 
+        navigationMapView?.inActiveNavigation = true
```

Comment:
Do we also need to add similar change for `CarPlayNavigationViewController`?

Reply:
Yes, it's added in the `CarPlayNavigationViewController ` after the active navigation starts.